openmetadata-ui/src/main/resources/ui/src/styles/app.less

```diff
@@ -197,3 +197,17 @@
   font-size: 1rem /* 16px */;
   line-height: 1.5rem /* 24px */;
 }
+
+/* SSO layout style  */
+.sso-container {
+  width: 100%;
+  height: 580px;
+  display: flex;
+  justify-content: space-evenly;
+  align-items: center;
+  flex-direction: column;
+
+  span {
+    margin: 0;
+  }
+}
```

Comment:
App.less is for utils classes and not for specific use case classes I would recommend creating `sso.less` or a similar file and use it, but not this!